lib/vtls/schannel.c

```diff
@@ -2754,6 +2754,148 @@ static void *schannel_get_internals(struct ssl_connect_data *connssl,
   return &backend->ctxt->ctxt_handle;
 }
 
+HCERTSTORE schannel_get_cached_cert_store(struct Curl_cfilter *cf,
```

Comment:
As this is not a static function, then it needs to be prefixed with `Curl_`.

Reply:
OK